media/admin/com_virtuemart/models/zasilkovna_src/VirtueMartModelZasilkovna/ShipmentMethod.php

```diff
@@ -380,5 +416,19 @@ public function getParams()
     {
         return $this->method;
     }
+
+    public function getShippingType()
+    {
+        return (string) $this->getParams()->shipping_type;
```

Comment:
Myslel bych si že shipping_type bude specifikovaný v zasilkovna.xml. Jak se tedy vykresluje to rozhraní?

Reply:
ano shipping_type je specifikovany v zasilkovna.xml, jen se mi to nejak nepridalo do commitu.